File: pinyougou_manager_web/src/main/webapp/js/service/brandService.js
//服务层
app.service('brandService',function ($http) {
    this.findAll=function () {
        return $http.get('../brand/findAll.do');
    };
    this.findPage=function (currentPage, pageSize) {
        return  $http.get('../brand/getPage.do?pageNum='+currentPage+'&pageSize='+pageSize)
    };
    this.findOne=function (id) {
        return  $http.get('../brand/findOne.do?id='+id)
    };
    this.add=function (entry) {
        return $http.post('../brand/save.do',entry)
    };
    this.update=function (entry) {
        return  $http.post('../brand/update.do',entry)
    };
    this.delete=function (ids) {
        return   $http.get('../brand/delete.do?ids='+ids)
    };
    this.search=function (pageNum, pageSize,searchEntry) {
        return $http.post('../brand/search.do?pageNum='+pageNum+'&pageSize='+pageSize,searchEntry)
    }
    //下拉列表数据
    this.selectOptionList=function () {
        return $http.get('../brand/selectOptionList.do');
    }
});

File: pingyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.entity.PageResult;
import com.pinyougou.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {
    List<TbBrand> findAllBrand();
   PageResult findPage(Integer pageNum, Integer pageSize);
    void save(TbBrand tbBrand);
    void update(TbBrand tbBrand);
    TbBrand findOne(Long id);
    void delete(Long []ids);
    PageResult findPage1(TbBrand tbBrand,Integer pageNum,Integer pageSize);

    /**
     * 返回下拉列表数据
     * @return
     */
    List<Map> selectOptionList();
}
